app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra'
+require './lib/takeaway'
+require './lib/text'
+
+$takeaway = Takeaway.new
+
+post '/sms' do
+
+  body = params['Body']
+
+  body.casecmp('menu').zero? ? send_menu : confirmation_text(body)
+
+end
+
+def confirmation_text(body)
+  $takeaway.parse_order(body)
+  total = $takeaway.order[0][:total]
```

Comment:
if possible define a better interface to the takeaway object with methods to return data you need rather than access data from keys from arrays held as state if that makes sense?